b2share/config.py

```diff
@@ -197,11 +378,10 @@
 
 B2ACCESS_APP_CREDENTIALS = dict(
     # B2ACCESS authentication key and secret
-    consumer_key=os.environ.get("B2ACCESS_CONSUMER_KEY"),
-    consumer_secret=os.environ.get("B2ACCESS_SECRET_KEY"),
+    consumer_key=os.environ.get("B2ACCESS_CONSUMER_KEY", '*** CONSUMER KEY ***'),
+    consumer_secret=os.environ.get("B2ACCESS_SECRET_KEY", '*** SECRET KEY ***'),
```

Comment:
`*** CONSUMER KEY ***` and `*** SECRET KEY ***` are just placeholders. If not defined flask will complain that values have not been set.